client/my-sites/hosting/sftp-card/ssh-key-loading-skeleton.tsx

```diff
@@ -0,0 +1,38 @@
+import { LoadingPlaceholder } from '@automattic/components';
+import styled from '@emotion/styled';
+
+const PlaceholderGroup = styled.div( {
+	display: 'flex',
+	flexDirection: 'column',
+	gap: '0.5em',
+} );
+
+const ParagraphPlaceholder = styled( LoadingPlaceholder )( {
+	width: 560,
+	maxWidth: '100%',
+} );
+const LinkPlaceholder = styled( ParagraphPlaceholder )( { width: 290, marginBottom: 30 } );
+const LabelPlaceholder = styled( ParagraphPlaceholder )( { width: 50 } );
+const InputPlaceholder = styled( ParagraphPlaceholder )( {
+	width: '80%',
+	height: 36,
+	marginBottom: 20,
+} );
+
+export function SshKeyLoadingSkeleton() {
```

Comment:
If this is for the `SftpCard`, should this be `SftpCardLoadingSkeleton`?

Reply:
done https://github.com/Automattic/wp-calypso/pull/73982/commits/02c4e377b90358b181a41950cab14e6db2e5b796